resources/assets/js/components/stekpakket/OptieWeergave.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+	<div>
+		<div class="kopje">{{ $parent.opties[index].groep }}</div>
+		<div class="opties" v-for="(details, key) in $parent.opties[index].opties">
+			<div class="selecteer">
+				<toggle-button
+					:width="40"
+					:value="$parent.opties[index].opties[key].actief"
+					@change="toggle($event, key)" :sync="true"/>
+			</div>
+			<div class="uitleg"
+				 @click="toggle({value: !$parent.opties[index].opties[key].actief}, key)">
+				{{ $parent.opties[index].opties[key].optie }}
+				<span>(&euro; {{ $parent.opties[index].opties[key].prijs.toFixed(2).replace('.', ',') }})</span>
+			</div>
+		</div>
+	</div>
+</template>
+<script lang="ts">
+	import Vue from 'vue';
+	import {ToggleButton} from 'vue-js-toggle-button';
+	import {Component, Prop} from 'vue-property-decorator';
+
+	@Component({
+		components: {ToggleButton},
+	})
+	export default class OptieWeergave extends Vue {
+		@Prop()
+		protected index: number;
+
+		protected toggle(value: any, key: string) {
+			if (this.$parent.laden) {
```

Comment:
Ik ben niet zo gecharmeerd van `$parent`, m'n PhpStorm weet ook niet wat ie er mee moet. deze kennis van boven mee geven is wat netter. Het zijn ook maar een paar dingen die meegegeven zouden moeten worden.

Reply:
Ik heb eerst een tijd zitten banjeren met het doorgeven van informatie via props en het updaten ervan via events, maar kreeg het niet voor elkaar om dan de toggles ook daadwerkelijk te laten updaten op het moment dat er iets in de parent veranderde. Vandaar de hacky oplossing